platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -619,7 +619,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.
+ like {field_name}.  (Token replacement is only supported for literal text-field
+ values--not for data-driven property functions.)
```

Comment:
Ah okay, I misunderstood: there _is_ a publicly documented way to refer to it.  It'll need to refer to both `MGLSourceStyleFunction` and `MGLCompositeStyleFunction`.